docs/source/actions.rst

```diff
@@ -0,0 +1,19 @@
+Portal Actions
+================
+
+You can get the available actions for a user on a context with the
```

Comment:
@csenger we usually have one or two sentences for devs who are not familiar with Plone telling them what this feature is about. We might also want to link to https://docs.plone.org/develop/plone/functionality/actions.html.

Reply:
I extended the documentation.